### ISSUE TITLE
Auto-validate websites

### ISSUE THREAD

**Issue by Lyla-Fischer** [AnanseGroup/atlas_of_innovation#41]
Make a crawler that validates the websites of all of the spaces and flags the ones where the website leads to an invalid domain.

**Comment by Lyla-Fischer** [comment]
@annawb the datafields doc says that we should also have the script validate email addresses. Do you mean that you want to send a yearly auto-email to revalidate email addresses? That isn't a common thing to see on the internet (invalid email addresses are just invalid)

**Comment by annawb** [comment]
Good point, this should only be about the website validation. We also want to email everyone yearly to check about spaces, but that’s not part of this trawler.
